web/packages/core/src/load-options.ts

```diff
@@ -117,6 +117,39 @@ export const enum WindowMode {
     Gpu = "gpu",
 }
 
+/**
+ * The render backend of a Ruffle player.
+ */
+export const enum RenderBackend {
+    /*
+     * A draft API that is currently unavailable, but will be preferred if available in the future.
```

Comment:
I do not wish to butt in on the PR, but, short time-specifying additions to the documentation comments might be helpful. Like for *"A draft API that is currently **as of April 2023** unavailable, but will .."*, since "currently" depends on the current time, and specifying at which year it was written may help avoid having to look it up for any readers. In my personal projects I have added similar dates to specify the time when I wrote "currently" in comments.

Reply:
Added the link. I will personally remind you guys to update the docs when we enable WebGPU.